spec/spec_helper.rb

```diff
@@ -51,3 +51,17 @@
 require 'zonebie/rspec'
 
 RSpec::Expectations.configuration.on_potential_false_positives = :nothing
+
+# Shared helper methods used in multiple files
```

Comment:
I am not convinced this is the right place for these, and it boggles my mind that I am the first person on the project to want to reuse a method between multiple tests, but I cannot find any example of putting these somewhere else?

Reply:
There are various flavors of spec helpers in `spec/support`, but I'd consider creating a new rspec matcher for something like this too.

Reply:
Some, but not all, are unintuitively included in all tests [here](https://github.com/18F/identity-idp/blob/4aa6aeb/spec/rails_helper.rb#L43-L49).

Reply:
Creating a new matcher to test the test for my new matcher feels like the turtles are starting to stack up a little too much.

I'm going to push an update moving this out of SpecHelper and into a new DiffHelper. Let me know if that seems like a reasonable enough approach.